test/accounts/ed25519.spec.ts

```diff
@@ -69,17 +69,4 @@ describe('ed25519 account', () => {
       expect(ec.id).to.eq('2bGCW3XbfLmSRhotYzcUgqiomiiFL7y3h7xPH8Ha3FVaYFp6YsjzSRSBeVjnae');
     });
   });
-
-  describe('#encryptSeed', () => {
-    it('should generate a cypher text from seed phrase', () => {
```

Comment:
Is this test not needed anymore?

Reply:
The code was not in the correct place. Plus the test was not good. We need to write more tests and will add this.